src/com/lilithsthrone/game/inventory/clothing/AbstractClothingType.java

```diff
@@ -1393,51 +1402,81 @@ public ClothingSet getClothingSet() {
 //		return SVGStringEquippedMap;
 //	}
 	
-	private void addSVGStringMapping(Colour colour, Colour colourSecondary, Colour colourTertiary, String s) {
+	private void addSVGStringMapping(Colour colour, Colour colourSecondary, Colour colourTertiary, String pattern, String s) {
+		if(pattern == null) {
+			pattern = "none"; // The map does not contain null as a key.
+		}
 		if(SVGStringMap.get(colour)==null) {
 			SVGStringMap.put(colour, new HashMap<>());
 			SVGStringMap.get(colour).put(colourSecondary, new HashMap<>());
+			SVGStringMap.get(colour).get(colourSecondary).put(colourTertiary, new HashMap<>());
 			
 		} else if(SVGStringMap.get(colour).get(colourSecondary)==null) {
 			SVGStringMap.get(colour).put(colourSecondary, new HashMap<>());
+			SVGStringMap.get(colour).get(colourSecondary).put(colourTertiary, new HashMap<>());
+		} else if(SVGStringMap.get(colour).get(colourSecondary).get(colourTertiary)==null) {
+			SVGStringMap.get(colour).get(colourSecondary).put(colourTertiary, new HashMap<>());
 		}
 		
-		SVGStringMap.get(colour).get(colourSecondary).put(colourTertiary, s);
+		SVGStringMap.get(colour).get(colourSecondary).get(colourTertiary).put(pattern, s);
 	}
 	
-	private void addSVGStringEquippedMapping(Colour colour, Colour colourSecondary, Colour colourTertiary, String s) {
+	private void addSVGStringEquippedMapping(Colour colour, Colour colourSecondary, Colour colourTertiary, String pattern, String s) {
+		if(pattern == null) {
+			pattern = "none"; // The map does not contain null as a key.
+		}
 		if(SVGStringEquippedMap.get(colour)==null) {
```

Comment:
duplicate code, combined with previous comment.... this is something to be handled separately I guess